sirepo/job_supervisor.py

```diff
@@ -887,6 +887,15 @@ async def _receive_api_runStatus(self, req):
             return self._init_db_missing_response(req)
         return r
 
+    async def _receive_api_sbatchLoginStatus(self, req):
+        r = False
+        o = self._create_op(job.OP_SBATCH_AGENT_READY, req)
+        try:
+            r = o.assign_driver().agent_is_ready_or_starting()
+        finally:
+            o.destroy()
+        return PKDict(ready=r)
```

Comment:
Maybe this should be loggedIn since everything else is now about login status?

Reply:
I'd like to leave it for now. It could change later.